src/OpenTelemetry.Exporter.Prometheus.HttpListener/PrometheusHttpListener.cs

```diff
@@ -19,36 +19,45 @@
 using System.Threading;
 using System.Threading.Tasks;
 using OpenTelemetry.Internal;
+using OpenTelemetry.Metrics;
 
-namespace OpenTelemetry.Exporter.Prometheus
+namespace OpenTelemetry.Exporter.Prometheus.HttpListener
 {
-    /// <summary>
-    /// An HTTP listener used to expose Prometheus metrics.
-    /// </summary>
-    internal sealed class PrometheusExporterHttpServer : IDisposable
+    public sealed class PrometheusHttpListener : IDisposable
     {
         private readonly PrometheusExporter exporter;
-        private readonly HttpListener httpListener = new();
+        private readonly System.Net.HttpListener httpListener = new();
```

Comment:
```suggestion
        private readonly HttpListener httpListener = new();
```

Reply:
Will get this waring without using System.Net for it :(.
What are the suggested ways to not spelling out the entire namespace for System.Net.HttpListener while not changing the PrometheusHttpListener namespace?
![image](https://user-images.githubusercontent.com/16979322/176331871-b55e274c-fe1e-4fa8-8965-6ab4d5b92f04.png)